File: result/translate/chatgpt/python2java/java/FIND_SUM_NODES_GIVEN_PERFECT_BINARY_TREE_1.java
static int f_gold(int l) {
    int leafNodeCount = (int) Math.pow(2, l - 1);
    int sumLastLevel = 0;
    sumLastLevel = ((leafNodeCount * (leafNodeCount + 1)) / 2);
    int sum = sumLastLevel * l;
    return sum;
}

File: result/translate/transcoder/java2python/python/FIND_SUM_NODES_GIVEN_PERFECT_BINARY_TREE_1.py
def f_gold ( l ) :
    leaf_node_count = np.arange ( 2 , l - 1 )
    sum_last_level = 0
    sum_last_level = ( leaf_node_count * ( leaf_node_count + 1 ) ) / 2
    sum = sum_last_level * l
    return sum
